versioned_docs/version-8.4/components/concepts/process-instance-migration.md

```diff
@@ -143,7 +143,6 @@ The following limitations exist that may be supported in future versions:
   - A process instance
   - A service task
 - The following scenarios cannot be migrated:
-  - A process instance with an incident
```

Comment:
❌ On 8.4, this limitation still exists. So we should keep this line.

Reply:
Ahh, correct, updated!